src/app/assets/icons/ClassBallDoubleDigit.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+type ClassBallProps = {
+  num: string;
+};
+
+export default function ClassBallDoubleDigit({
+  num,
+}: ClassBallProps): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="25"
+      height="25"
+      viewBox="0 0 25 25"
+    >
+      <g
+        fill="var(--primary-color)"
+        stroke="var(--primary-color)"
+        stroke-width="1"
+      >
+        <circle cx="12.5" cy="12.5" r="12.5" stroke="none" />
+        <circle cx="12.5" cy="12.5" r="12" fill="none" />
+      </g>
+      <text
+        transform="translate(12 19)"
+        fill="var(--contrast-text-color)"
+        font-size="16"
+        font-family="Roboto-Regular, Roboto"
+        letter-spacing="0.025em"
+      >
+        <tspan x="-9.184" y="0">
+          {num}
+        </tspan>
+      </text>
```

Comment:
This is the difference to `ClassBallNo` and `ClassBallSingleDigit`.

You can reduce all three icons to a single icon to reduce duplicate code if you add [conditional rendering](https://reactjs.org/docs/conditional-rendering.html):

```suggestion
      {digit && <text
        transform="translate(12 19)"
        fill="var(--contrast-text-color)"
        font-size="16"
        font-family="Roboto-Regular, Roboto"
        letter-spacing="0.025em"
      >
        <tspan x={digit > 9 ? -9.184 : -4.492 } y="0">
          {num}
        </tspan>
      </text>
      } 
```

Reply:
Yes, will do. Will setup a separate issue for this. I have some more conditional rendering topics on may pages and I planned it for the start of next week. For now, I will continue with the status quo, knowing, that it still needs to be improved. Thank you.